jobs/JGDAS_FIT2OBS

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "fit2obs" -c "base fit2obs"
+
+
+##############################################
+# Set variables used in the script
+##############################################
+
+export CDATE=${CDATE:-${PDY}${cyc}}
+export CDUMP=${CDUMP:-${RUN:-"gdas"}}
+
+export xdate=$(${NDATE} -${VBACKUP_FITS} ${CDATE})
+export CDATE=${xdate} # Reset CDATE to xdate for lookback
+export vday=$(echo ${xdate} | cut -c1-8)
+export vcyc=$(echo ${xdate} | cut -c9-10)
+
+export COM_INA=${ROTDIR}/gdas.${vday}/${vcyc}/atmos
+export COM_INF='$ROTDIR/vrfyarch/gfs.$fdy/$fzz'
+export COM_PRP=${ROTDIR}/gdas.${vday}/${vcyc}/obs
+
+export PRPI=${COM_PRP}/${CDUMP}.t${vcyc}z.prepbufr
+export sig1=${COM_INA}/${CDUMP}.t${vcyc}z.atmanl.nc
+export sfc1=${COM_INA}/${CDUMP}.t${vcyc}z.atmanl.nc
```

Comment:
Is this a mistake?
```suggestion
export sfc1=${COM_INA}/${CDUMP}.t${vcyc}z.sfcanl.nc
```

Reply:
This was lifted right from the Fit2Obs scripts: https://github.com/NOAA-EMC/Fit2Obs/blob/develop/batrun/excfs_gdas_vrfyfits.sh.ecf#L64
It may be a mistake in Fit2Obs package. @jack-woollen should the `sfc1` file be the `sfcanl.nc` file instead of the `atmanl.nc` file?

Reply:
@KateFriedman-NOAA Well its not exactly correct, but a do no harm workaround. I need to write code to process the netcdf sfcanl, not a high priority at the moment.